exporter/context.go

```diff
@@ -109,9 +109,7 @@ func newImportContext(c *common.DatabricksClient) *importContext {
 			{regexp.MustCompile(`\W+`), ""},
 			{regexp.MustCompile(`[_]{2,}`), "_"},
 		},
-		hclFixes: []regexFix{
-			{regexp.MustCompile(`\{ `), "{\n\t\t"},
-			{regexp.MustCompile(`, `), ",\n\t\t"},
+		hclFixes: []regexFix{ // Be careful with that! it may break working code
```

Comment:
ew... code will be ugly.. but okay.

Reply:
it was really breaking the spark-submit parameters :-(